deployability/modules/workflow_engine/examples/manager/vagrant/dtt1-managers-poc-vagrant.yaml

```diff
@@ -49,32 +42,12 @@ tasks:
           - modules/testing/main.py
           - targets:
             - wazuh-1: "{working-dir}/manager-linux-centos-7-amd64/inventory.yaml"
-            - wazuh-2: "{working-dir}/manager-linux-ubuntu-20.04-amd64/inventory.yaml"
-            - wazuh-3: "{working-dir}/manager-linux-ubuntu-22.04-amd64/inventory.yaml"
-            - wazuh-4: "{working-dir}/manager-linux-oracle-9-amd64/inventory.yaml"
-            - wazuh-5: "{working-dir}/manager-linux-amazon-2-amd64/inventory.yaml"
-            - wazuh-6: "{working-dir}/manager-linux-redhat-7-amd64/inventory.yaml"
-            - wazuh-7: "{working-dir}/manager-linux-redhat-8-amd64/inventory.yaml"
-            - wazuh-8: "{working-dir}/manager-linux-redhat-9-amd64/inventory.yaml"
-            - wazuh-9: "{working-dir}/manager-linux-centos-8-amd64/inventory.yaml"
-            - wazuh-10: "{working-dir}/manager-linux-debian-10-amd64/inventory.yaml"
-            - wazuh-11: "{working-dir}/manager-linux-debian-11-amd64/inventory.yaml"
-            - wazuh-12: "{working-dir}/manager-linux-debian-12-amd64/inventory.yaml"
-          - tests: "install,restart,stop,uninstall"
+            - wazuh-2: "{working-dir}/manager-linux-centos-8-amd64/inventory.yaml"
+          - tests: "install"
           - component: "manager"
           - wazuh-version: "4.7.4"
           - wazuh-revision: "40717"
-          - live: "True"
     depends-on:
-      - "allocate-manager-linux-ubuntu-20.04-amd64"
-      - "allocate-manager-linux-ubuntu-22.04-amd64"
-      - "allocate-manager-linux-amazon-2-amd64"
-      - "allocate-manager-linux-redhat-7-amd64"
-      - "allocate-manager-linux-redhat-8-amd64"
-      - "allocate-manager-linux-redhat-9-amd64"
       - "allocate-manager-linux-centos-7-amd64"
       - "allocate-manager-linux-centos-8-amd64"
-      - "allocate-manager-linux-debian-10-amd64"
-      - "allocate-manager-linux-debian-11-amd64"
-      - "allocate-manager-linux-debian-12-amd64"
-      - "allocate-manager-linux-oracle-9-amd64"
```

Comment:
Restore them

Reply:
Done in [a7aa606](https://github.com/wazuh/wazuh-qa/pull/5392/commits/a7aa6062b7d70d81d64975f698f29e7d4959c353)